examples/LAPACK-CBLAS/config.cmake

```diff
@@ -0,0 +1,3 @@
+include(hunter_cmake_args)
+
+hunter_cmake_args(LAPACK CMAKE_ARGS CBLAS=ON)
```

Comment:
It should be `hunter_config`, not `hunter_cmake_args`:
* https://github.com/ruslo/hunter/wiki/example.custom.config.id#custom-cmake-options